### PR TITLE
Make use of doxygen todo

### PR DESCRIPTION
**Pull request by cgundogan**
This PR replaces some TODO comments that I found with the appropriate `@todo` doxygen annotation in order to list them in the doc's `Todo List` section.

### CODE REVIEW

**Comment by kaspar030** [comment]
hm, not changing legacy network code would save me some rebasing conflicts...

**Comment by cgundogan** [comment]
@kaspar030 sure, İ understand. Could you briefly tell me directly which ones I should remove from this PR?

File: sys/include/net/ng_icmpv6/error.h [hunk 1]
```diff
@@ -16,6 +16,8 @@
  * @brief   ICMPv6 error message definitions
  *
  * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * @todo implement build and handle functions
```
Comment (authmillenon) [inline sys/include/net/ng_icmpv6/error.h]: See #3184.

**Comment by cgundogan** [comment]
@kaspar030 I kicked out the changes to `tcp`, `nhdp` and `socket_base`.
@authmillenon while #3184 is still unmerged I do not intend to make any changes.

**Comment by kaspar030** [comment]
@cgundogan thx!

**Comment by authmillenon** [comment]
> @authmillenon while #3184 is still unmerged I do not intend to make any changes.

kk. Then ACK from me, if travis is happy.

**Comment by cgundogan** [comment]
@authmillenon travis is happy

**Comment by authmillenon** [comment]
Don't dare to push the merge button yourself ;-P

**Pull request merged**